### ISSUE TITLE
PHP Warning call_user_func() expects parameter 1 to be a valid callback, no array or string given 

### ISSUE THREAD

**Issue by mikebronner** [wpovernight/woocommerce-pdf-invoices-packing-slips#71]
Got the following error after installing:

plugin: 2.4.3
wordpress: 5.3.2
WooCommerce: 3.9
php: 7.4

```
PHP Warning call_user_func() expects parameter 1 to be a valid callback, no array or string given 
    [internal] call_user_func
    /home/forge/imaginethatcomics.com/releases/20200128015254/public/app/plugins/woocommerce-pdf-invoices-packing-slips/includes/class-wcpdf-setup-wizard.php:241 WPO\WC\PDF_Invoices\Setup_Wizard::save_step
    /home/forge/imaginethatcomics.com/releases/20200128015254/public/app/plugins/woocommerce-pdf-invoices-packing-slips/includes/class-wcpdf-setup-wizard.php:105 WPO\WC\PDF_Invoices\Setup_Wizard::setup_wizard
    /home/forge/imaginethatcomics.com/releases/20200128015254/public/wordpress/wp-includes/class-wp-hook.php:288 WP_Hook::apply_filters
    /home/forge/imaginethatcomics.com/releases/20200128015254/public/wordpress/wp-includes/class-wp-hook.php:312 WP_Hook::do_action
    /home/forge/imaginethatcomics.com/releases/20200128015254/public/wordpress/wp-includes/plugin.php:478 do_action
    /home/forge/imaginethatcomics.com/releases/20200128015254/public/wordpress/wp-admin/admin.php:170 [main]
```

**Comment by Spreeuw** [comment]
Thanks for reporting, this was fixed in v2.4.4, by 9934b12754b853bc1a37983a8c4c091d15eb38b9. Can you confirm this (since you reported this today but do mention v2.4.3 in your report)?

**Comment by mikebronner** [comment]
@Spreeuw Thanks, I just installed 2.4.4 after reporting this. Will close for now, and re-open if it recurs. :)
